### PR TITLE
send otp to email

### PR DESCRIPTION
**Pull request by JRakin**

### CODE REVIEW

**Comment by JRakin** [comment]
@sifatul vaiya everything is done so far but i have to test it several times and have to check the corner cases, please review this pr in the meantime i'll test and if anything is left i'll update

File: Utils/mongodb.tsx [hunk 1]
```diff
@@ -25,13 +25,15 @@ export async function connectToDatabase() {
         useUnifiedTopology: true,
         serverApi: ServerApiVersion.v1,
     };
-    const client = new MongoClient(uri, options);
+    const client = await mongoose.connect(uri, options);
```
Comment (sifatul) [inline Utils/mongodb.tsx:28]: @JRakin bhai aita merge korle baki api gula kaj kora bondho kore dibe.

Comment (JRakin) [reply Utils/mongodb.tsx:28]: @sifatul vaiya change kore dibo

File: components/form/otpVerificationForm.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import { Box, Button, TextField } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import * as React from 'react';
+
+function OtpVerificationForm() {
+    return (
+        <Box
+            sx={{
+                display: 'flex',
+                justifyContent: 'center',
```
Comment (sifatul) [inline components/form/otpVerificationForm.tsx:10]: @JRakin can we not use inline css.

**Review by sifatul — APPROVED** [review]

File: components/form/otpVerificationForm.tsx (hunk 2 above)
Comment (JRakin) [reply components/form/otpVerificationForm.tsx:10]: @sifatul vaiya this will all be removed later when Ibrahim vai will be finished creating the reusable component I just designed a minimal UI

**Comment by sifatul** [comment]
@fazlerabby07 bhai please review and feel free to merge
